templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
+  <div class="e-commerce-ampstart-hero center">
+    <div class="e-commerce-ampstart-hero-img">
+      {{#images}}
+        {{#image}}
+          <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="responsive" media="{{media}}" alt="{{alt}}"></amp-img>
+        {{/image}}
+      {{/images}}
+    </div>
+    <div class="e-commerce-ampstart-hero-content e-commerce-ampstart-hero-theme-{{text-theme}} flex justify-center flex-column">
+      <div>
+      <h2 class="e-commerce-ampstart-hero-title h3 caps">{{title}}</h2>
+      <p class="e-commerce-ampstart-hero-body">{{body}}</p>
+      {{#link}}
+        <button class="ampstart-btn ampstart-btn-secondary caps">{{text}}</button>
```

Comment:
if you want this to be a link use the `<a>` tag if this is a button change the json - if it navigates to a new page or different section on the same page then it is a link